pkg/gui/controllers/branches_controller.go

```diff
@@ -557,6 +557,7 @@ func (self *BranchesController) delete(branches []*models.Branch) error {
 		Label: lo.Ternary(len(branches) > 1, self.c.Tr.DeleteLocalBranches, self.c.Tr.DeleteLocalBranch),
 		Key:   'c',
 		OnPress: func() error {
+			self.context().SetSelection(startIdx)
```

Comment:
I'd prefer to do this only after deleting succeeded; if deleting the branches fails for whatever reason, the selection should stay unchanged. I pushed a fixup to change this (362a8f949673).